[docs] Update Error Knowledge Base (#6136)
conan-io/conan-center-index: docs/error_knowledge_base.md

#### **<a name="KB-H053">#KB-H053</a>: "PRIVATE IMPORTS"**

The recipe imports private Conan API, this is strongly discouraged - private imports are subjects to breaking changes. Avoid usage of private APIs, request to publically expose needed methods, if necessary.

#### **<a name="KB-H054">#KB-H054</a>: "LIBRARY DOES NOT EXIST"**

Libraries which are listed on [Components](https://docs.conan.io/en/latest/creating_packages/package_information.html#package-information-components) must be present in `libdirs`. Check if the library name is correct, or if a library is only generated for a specific platform.

#### **<a name="KB-H055">#KB-H055</a>: "SINGLE REQUIRES"**

> Both `requirements()` and `self.requires` should not be used together at same recipe.

Reviewer: ```suggestion
Do not use `requirements()` and `self.requires` together in the same recipe.
```

Author: Nice
